bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CommonWidgetsDPIChangeHandlers.java

```diff
@@ -11,9 +11,10 @@
  * Contributors:
  *     Yatta Solutions - initial API and implementation
  *******************************************************************************/
-package org.eclipse.swt.internal;
+package org.eclipse.swt.custom;
```

Comment:
That file must stay in the win32 project, so please move it to ..\bundles\org.eclipse.swt\Eclipse SWT\win32 creating a new org.eclipse.swt.custom package there.